### PR TITLE
DDPB-3697 rightsize redis

### PR DESCRIPTION
**Pull request by jamesrwarren**
## Purpose
This should just work. After running the behat tests locally and looking at the keys stored in redis it all seems to be:

image

Both of these should be unique. The short one is actually based on userid it seems which is unique in the user table.

image

As such this all should just work fine.

EDIT: This didn't work just fine. If session is being stored across admin and frontend like behat does when running certain tests then it can cause variables to not be set. The solution is to add admin and front to the prefixes so then it will not use session from the other app type as it's own.

Fixes DDPB-3697

## Approach
Changed the local config. Checked everything still worked. Updated terraform.

## Learning
NA

## Checklist
- [x] I have performed a self-review of my own code
- [ ] I have updated documentation (Confluence/ADR/tech debt doc) where relevant
- [ ] I have added tests to prove my work
- [ ] The product team have approved these changes

### Frontend
- [ ] I have run an in-browser accessibility test (e.g. WAVE, Lighthouse)
- [ ] There are no deprecated CSS classes noted in the profiler
- [ ] Translations are used and the profiler doesn't identify any missing
- [ ] Any links or buttons added are screen reader friendly and contextually complete

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -81,6 +82,12 @@ reset-database: ## Resets the DB schema and runs migrations
 reset-fixtures: ## Resets the DB contents and reloads fixtures
 	docker-compose run --rm api sh scripts/reset_db_fixtures_local.sh
 
+redis-clear: ## Puts app in dev mode and disables debug (so the app runs faster, but no toolbar/profiling)
```
Comment (acsauk) [inline Makefile:85]: The description here needs an update/removal

File: environment/checklist_sync.tf [hunk 2]
```diff
@@ -122,9 +122,10 @@ locals {
       { "name": "S3_BUCKETNAME", "value": "pa-uploads-${local.environment}" },
       { "name": "APP_ENV", "value": "${local.account.app_env}" },
       { "name": "OPG_DOCKER_TAG", "value": "${var.OPG_DOCKER_TAG}" },
-      { "name": "ADMIN_HOST", "value": "https://${aws_route53_record.admin.fqdn}" },
+      { "name": "ADMIN_HOST", "value": "https://${aws_route53_record.frontend_redis.fqdn}" },
```
Comment (acsauk) [inline environment/checklist_sync.tf:125]: Should this remain as admin?

File: environment/document_sync.tf [hunk 3]
```diff
@@ -118,9 +118,10 @@ locals {
       { "name": "S3_BUCKETNAME", "value": "pa-uploads-${local.environment}" },
       { "name": "APP_ENV", "value": "${local.account.app_env}" },
       { "name": "OPG_DOCKER_TAG", "value": "${var.OPG_DOCKER_TAG}" },
-      { "name": "ADMIN_HOST", "value": "https://${aws_route53_record.admin.fqdn}" },
+      { "name": "ADMIN_HOST", "value": "https://${aws_route53_record.frontend_redis.fqdn}" },
```
Comment (acsauk) [inline environment/document_sync.tf:121]: Same as checklist, should this still be admin?

**Review by acsauk — COMMENTED** [review]
A couple of Qs on same var values but awesome job - should be a nice saving. Incidentally, you may have fixed an issue I was having with some behat fixture steps - will pull this branch in and see!

File: environment/checklist_sync.tf (hunk 2 above)
Comment (jamesrwarren) [reply environment/checklist_sync.tf:125]: omg yes!

**Review by acsauk — APPROVED** [review]
:shipit:

**Pull request merged**